aries_vcx/src/handlers/connection/public_agent.rs

```diff
@@ -43,6 +43,10 @@ impl PublicAgent {
         self.institution_did.clone()
     }
 
+    pub fn service(&self) -> VcxResult<FullService> {
```

Comment:
Not being used - luckly :-D Because it wouldn't make much sense to me. The service resolved from ledger might actually refer to a different, newer, public agent.

Reply:
Oh I see it's actually used from `libvcx`. 

if I get it right this should return service associated to the given public agent, but as pointed out above, this won't be the case if the data is resolved from ledger. The `self.institution_did` might have written arbitrary value for `service` field on ledger and it won't necessary match `self` public agent.

Reply:
The agent's service is now implied from pairwise and cloud agent info instead from ledger now.